modules/apps/style-book/style-book-api/src/main/java/com/liferay/style/book/exception/ThemeIdNotFoundStyleBookEntryFileException.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.style.book.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Anderson Luiz
+ */
+public class ThemeIdNotFoundStyleBookEntryFileException
+	extends PortalException {
+}
```

Comment:
No, but I see many that has the methods, but they are not being used, i.e: `InvalidContactsCardTemplateSizeException`, `NoSuchContactsCardTemplateException`, `NoSuchContactsLayoutTemplateException`, `AssetLinkException`...

I just remove them since its not being used, for now

Reply:
You must specify the exception in the service.xml file ([example](https://github.com/liferay/liferay-portal/blob/5a2c0e3dbaf51c8a0e9f44826558654ecbbabedc/modules/apps/style-book/style-book-service/service.xml#L57)). Even though we have constructors without usages, we must keep them just for the sake of "Brian will complain". I proposed bringing this to broader discussion in the backend channels and tagging whoever might be responsible to decide this.

Reply:
I've raised a thread about the exceptions patterns on backend-dev. Tagged both of you.

Reply:
I added back, just for consistency with the others